sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ if (NOT DEFINED UNIFIED_RUNTIME_LIBRARY OR NOT DEFINED UNIFIED_RUNTIME_INCLUDE_D
   set(CMAKE_CXX_FLAGS_BAK "${CMAKE_CXX_FLAGS}")
   if (UNIX)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-error")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-suggest-override")
```

Comment:
Why -Wno-error above is not enough? It was meant to ignore all warnings when built from SYCL

Reply:
See https://github.com/intel/llvm/pull/8683 where we are trying to fix UR build

Reply:
I had a build failure I thought was caused by this warning in loader.  Checking again, this isn't needed. 

I removed it and rebased this on #8683